tests/integration/new-integration-test.sh

```diff
@@ -0,0 +1,197 @@
+set -eo pipefail
```

Comment:
Why no -u?

Reply:
then the color coding for `SUCCESS` and `FAILURE` don't work